dynamodb/src/main/scala/monix/connect/dynamodb/DynamoDbSubscriber.scala

```diff
@@ -18,44 +18,64 @@
 package monix.connect.dynamodb
 
 import monix.eval.Task
+import monix.execution.Ack.Continue
+import monix.execution.atomic.AtomicInt
 import monix.execution.cancelables.AssignableCancelable
 import monix.execution.{Ack, Callback, Scheduler}
-import monix.reactive.observers.Subscriber
 import monix.reactive.Consumer
 import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
 import software.amazon.awssdk.services.dynamodb.model.{DynamoDbRequest, DynamoDbResponse}
+import monix.reactive.observers.{SafeSubscriber, Subscriber}
 
+import scala.compat.java8.FutureConverters._
 import scala.concurrent.Future
 
-private[dynamodb] class DynamoDbSubscriber[In <: DynamoDbRequest, Out <: DynamoDbResponse]()(
+private[dynamodb] class DynamoDbSubscriber[In <: DynamoDbRequest, Out <: DynamoDbResponse](retriesPerOp: Int)(
   implicit
   dynamoDbOp: DynamoDbOp[In, Out],
   client: DynamoDbAsyncClient)
   extends Consumer[In, Out] {
 
+  require(retriesPerOp >= 1, "Retries per operation must be higher or equal than 1.")
+
   override def createSubscriber(cb: Callback[Throwable, Out], s: Scheduler): (Subscriber[In], AssignableCancelable) = {
-    val sub = new Subscriber[In] {
+    val safeSubscriber = SafeSubscriber {
```

Comment:
Why `SafeSubscriber` - did you notice any bugs, or are you not confident about following the protocol?

Reply:
No, guess I was just thinking on making it safe, since the last implementation wasn't [see bug ticket], but in this case there is no need to use `SafeSubscriber`.